x-pack/plugins/security_solution/server/lib/detection_engine/signals/threat_mapping/enrich_signal_threat_matches.ts

```diff
@@ -52,14 +52,15 @@ export const buildEnrichments = ({
   queries.map((query) => {
     const matchedThreat = threats.find((threat) => threat._id === query.id);
     const indicatorValue = get(matchedThreat?._source, indicatorPath) as unknown;
+    const feed = (get(matchedThreat?._source, FEED_PATH) ?? {}) as Record<string, unknown>;
     const indicator = ([indicatorValue].flat()[0] ?? {}) as Record<string, unknown>;
     if (!isObject(indicator)) {
       throw new Error(`Expected indicator field to be an object, but found: ${indicator}`);
     }
     const atomic = get(matchedThreat?._source, query.value) as unknown;
-
     return {
       indicator,
+      feed,
```

Comment:
I thought we only wanted and mapped the `threat.feed.name` field for the alert, so I think we could consider just putting feed.name here instead of the feed obj

Reply:
Agreed.